src/sources/internal_metrics.rs

```diff
@@ -66,12 +66,13 @@ async fn run(
 
         let metrics = capture_metrics(controller);
 
-        out.send_all(&mut stream::iter(metrics.map(|mut metric| {
+        out.send_all(&mut stream::iter(metrics).map(|mut metric| {
             if let Ok(hostname) = &hostname {
-                metric.insert_tag("host".into(), hostname.into());
+                metric.insert_tag(log_schema().host_key().to_owned(), hostname.to_owned());
             }
+            metric.insert_tag(String::from("pid"), std::process::id().to_string());
```

Comment:
The PID is not going to be different on each metric. Best to produce the string outside of the loop and just clone it here.

Reply:
Good catch. Adjusted in 299e079f2